src/Components/Sidebar/sidebar.css

```diff
@@ -0,0 +1,70 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+*{
+    padding: 0;
+    margin: 0;
+    /* box-sizing: border-box; */
+}
+body{
+    font-family: 'Lato', sans-serif;
+}
+.main-wrapper{
+    display: flex;
+}
+/* Sidebar */
+.sidebar{
+    position: fixed;
+    flex-grow: 0;
+    top: 5rem;
+    height: 100%;
+    border-right: 2px solid black;
+    padding: 1.25rem;
```

Comment:
Suggestion : For spacings you can make use of variables to have consistency throughout the code.